### ISSUE TITLE
Wrong speed

### ISSUE THREAD

**Issue by g5-freemen** [Quake4/MindMiner#51]
Something goes wrong? :)

![image](https://user-images.githubusercontent.com/36886126/39398905-792af69a-4b1d-11e8-8217-bc5b2291f67a.png)

**Comment by Quake4** [comment]
Miner window please

**Comment by Quake4** [comment]
No, this switch dont hase effect. Just disable this algo in this miner.
